### PR TITLE
[python] Task condition missing two downstream param

### LINKED ISSUE #7763: [Bug] [python] Task condition missing two downstream parameters
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

For now, our condition task only set upstream node, but without set downstream. It's a mistake and condition task have three required parameters, `ConditionOperator`, `downstream`, `upstream`. So we should add these two missing parameters

### What you expected to happen

Should also add condition success and failed downsteam

### How to reproduce

Run `python ./example/task_condition_example.py`

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhongjiajie**
We add two downstream tasks to set task condition
success and failed node

close: #7763

### CODE REVIEW

**Comment by zhongjiajie** [comment]
@devosend Do you have time to take a look at this PR?

**Comment by devosend** [comment]
yeah, I will review this later today.

File: dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py [hunk 1]
```diff
@@ -182,4 +201,5 @@ def task_params(self, camel_attr: bool = True, custom_attr: set = None) -> Dict:
         """
         params = super().task_params
         params["dependence"] = self.condition.get_define()
+        params["conditionResult"] = self.condition_result
```
Comment (devosend) [inline dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py]: Is this line redundant? beause `condition_result ` defined in custom_attr of `Task.task_params`

File: dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/task.py [hunk 2]
```diff
@@ -156,7 +157,8 @@ def __init__(
         self.resource_list = resource_list or []
         self.dependence = dependence or {}
         self.wait_start_timeout = wait_start_timeout or {}
-        self.condition_result = condition_result or self.DEFAULT_CONDITION_RESULT
+        if task_type != TaskType.CONDITIONS:
+            self.condition_result = condition_result or self.DEFAULT_CONDITION_RESULT
```
Comment (devosend) [inline dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/task.py]: Maybe you can also define a function decorated with `@condition_result.setter ` in `Conditions` task.

File: dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py (hunk 1 above)
Comment (zhongjiajie) [reply dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py]: Cause we can not set property `property` in class `Conditions` So I overwrite function `task_params`. It related your above comment https://github.com/apache/dolphinscheduler/pull/7783/files/2dd5134de92cca727514907e1863ab1db523055c#r778015716 .

Comment (zhongjiajie) [reply dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py]: I would try whether it work when I add setter in class `Task` attribute `condition_result`

Comment (zhongjiajie) [reply dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/condition.py]: I add property condition_result getter and setter in 169dd13, PTAL, thanks

**Comment by zhongjiajie** [comment]
@devosend I add property condition_result getter and setter in 169dd13, PTAL, thanks

**Review by devosend — COMMENTED** [review]
LGTM. good job! @zhongjiajie

**Comment by zhongjiajie** [comment]
> LGTM. good job! @zhongjiajie

Thanks for viewing 👍

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**